test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -891,7 +891,7 @@ define([
                         error_message: 'Something went terribly wrong',
                         error_code: 'throw new Error("ZOMG! NOOO!")',
                     };
-                    runLaunchErrorTest(this);
+                    return runLaunchErrorTest(this);
```

Comment:
Not returning the Promise was causing some occasional test issues. Glad it was an easy fix! 😌